tst/benchmarking/benchmark_commons/test_hpo_main_local.py

```diff
@@ -50,27 +51,37 @@ def test_tuner_is_run_the_expected_number_of_times(
             "nas201-ImageNet16-120": nas201_mo_benchmark("ImageNet16-120"),
         }
 
+        config_space = {
```

Comment:
Let us at least use the correct space for these benchmarks:

```
    hp_cols = ["hp_x0", "hp_x1", "hp_x2", "hp_x3", "hp_x4", "hp_x5"]

    config_space = {
        node: choice(
            ["avg_pool_3x3", "nor_conv_3x3", "skip_connect", "nor_conv_1x1", "none"]
        )
        for node in hp_cols
    }
    config_space["epochs"] = 200
```

Reply:
Updated